Vagrantfile

```diff
@@ -1,3 +1,58 @@
+
+
+targets = {
+
+  #
+  # the following targets are used with local vagrant providers (i.e virtualbox). 
+  #
+
+  "centos6.5" => { "box" => "chef/centos-6.5" },
+  "centos7"   => { "box" => "chef/centos-7.0" },
+  "ubuntu14"  => { "box" => "ubuntu/trusty64" },    
+  "ubuntu12"  => { "box" => "ubuntu/precise64" },
+  "freebsd10" => { "box" => "chef/freebsd-10.0" },
+
+  #
+  # the following targets are used by the aws vagrant provider. 
+  #
+
+  "aws-amazon2015.03" => { 
+    "box" => "andytson/aws-dummy",
+    "regions" => { "us-east-1" => "ami-1ecae776", "us-west-1" => "ami-d114f295" },
+    "username" => "ec2-user"
+  },
+  "aws-rhel7.1" => { 
+    "box" => "andytson/aws-dummy",
+    "regions" => { "us-east-1" => "ami-12663b7a", "us-west-1" => "ami-a540a5e1" },
+    "username" => "ec2-user"
+  },
+  "aws-rhel6.5" => {
+    "box" => "andytson/aws-dummy", 
+    "regions" => { "us-east-1" => "ami-1643ff7e", "us-west-1" => "ami-2b171d6e" },
+    "username" => "ec2-user"
+  },
+  "aws-centos7" => { 
```

Comment:
having the aws-based targets allows devs to spin up multiple vms w/o consuming local resources on their laptop. i have no problem removing the dups if folks prefer that.